folium/plugins/mouse_position.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+import json
+
+from branca.element import CssLink, Figure, JavascriptLink, MacroElement
+
+from jinja2 import Template
+
+
+class MousePosition(MacroElement):
+    """
+    Adds a mouse position widget on the map.
+
+    Parameters
+    ----------
+    position: location of the widget
```

Comment:
Docstring convention is name, type and default value on the first line, description on the second. So something like:

```
position : str, default 'bottomright'
    Location of the widget.
```